sdk/src/main/scala/com/horizen/api/http/SidechainTransactionApiRoute.scala

```diff
@@ -175,7 +175,9 @@ case class SidechainTransactionApiRoute(override val settings: RESTApiSettings,
       applyOnNodeView { sidechainNodeView =>
         val wallet = sidechainNodeView.getNodeWallet
         val inputBoxes = wallet.allBoxes().asScala
-          .filter(box => body.transactionInputs.exists(p => p.boxId.contentEquals(BytesUtils.toHexString(box.id()))))
+          .filter(box =>
+            (box.isInstanceOf[ZenBox] || box.isInstanceOf[ForgerBox]) &&
```

Comment:
why do we filter with ZenBox and ForgerBox only?
Actually we can spend any box locked by PublicKey25519Proposition.

Reply:
we are creating a core transaction here: are'nt the only two types of boxes we can have in input in this tx?

Reply:
was agreed to keep only ZenBoxes and ForgerBoxes available.